### ISSUE TITLE
Two expanded ascii codes are printed on the screen during the display of Turkish characters (ı, İ, ü, Ü, ö, Ö, Ş, Ş, ğ, Ğ, ç, Ç).

### ISSUE THREAD

**Issue by PozitronikTech** [freetronics/DMD2#54]
utf-8 extended ascii me dec (195,156) or hex (C3.9C) return extended ascii code of 'ü' character dec (252) or hex (FC). 
The dmd library produces individual characters for each incoming code of the incoming character and prints them on the screen. For the 'ü' character, the characters on the screen with two dec (195,156) print the screen.
In the 'a' character, the second code is 0. It comes as dec (97,0). 
In other words, the second dec value of each ascii code between 0-127 is 0. 
Is it possible to make an arrangement for those whose second dec value is greater than zero by making an arrangement in the code part? 
Site with extended ascii table
https://www.rapidtables.com/code/text/ascii-table.html
I hope my english was successful. Thank u.
